### ISSUE TITLE
[Feature] Generate QR code

### ISSUE THREAD

**Issue by sumitsum** [appsmithorg/appsmith#4175]
## Summary
Provide a feature to generate QR code.

## Additional Context
```
Hey Appsmith-Team,
I have some questions for you.
Is it possible to generate qr codes within appsmith?
```

**Comment by areyabhishek** [comment]
@sumitsum is it an option to use a service to generate QR codes and display them as an image in the image widget?
What would they want to generate QR codes for? Could you find a few more details?

**Comment by sumitsum** [comment]
Hi @areyabhishek, user wants to generate the QR code and display it in the image widget: 
`I want to be able to convert the address-string (e.g. "d49931814e6bfbb8e221f1b2936e206e5a6b6abeb9c070f8603814c525c9abe3") into a qr-code and display it in a img-widget like @Abhishek Nayak suggested on the github-issue.`

**Comment by Nikhil-Nandagopal** [comment]
@sumitsum I believe abhishek is suggesting that you can ask the user to call an API like this
https://www.qr-code-generator.com/qr-code-api/
which will return a URL and then the user can bind that to the Image widget

**Comment by sumitsum** [comment]
@Nikhil-Nandagopal Sorry, I misunderstood. Got it now. I will respond to the user with this suggestion.

**Comment by somangshu** [comment]
@Nikhil-Nandagopal @areyabhishek Should this instead be the ability to scan a QR code and connect to an API for fetching some record from the DB/Datasource?

**Comment by Nikhil-Nandagopal** [comment]
@somangshu this particular case is about a special image widget that takes in a text and displays a QR code. It's an encoding solution.
The case you are talking about is also valid but can be an action which triggers a camera and decodes the QR image back to text. i.e a decoding solution

**Comment by somangshu** [comment]
@Nikhil-Nandagopal  Yes, There was a user who wants to build a QC app for their logistics chain and wanted such an option to quickly access the SKU which needs to be QC'ed. Should this be raised as a separate issue?

**Comment by Nikhil-Nandagopal** [comment]
@somangshu yeah separate issue please

**Comment by st-trade** [comment]
Hi! My use case is the following.
A first user shows a QR that represents certain article which is an entry in a table so a second user with a different role can scan that qr and validate that article and perform actions in their end.

**Comment by dhruvikn** [comment]
Whenever prepping the QR Code Generator for production, need to take care of these issues - https://github.com/appsmithorg/appsmith/pull/15990#discussion_r962672349

cc: @dilippitchika @somangshu
